styles/deepl/catppuccin.user.css

```diff
@@ -72,675 +70,283 @@
+    #top-navi-sidemenu-opener,
+    .pageFooterV2-module--socialMediaLinks--71305,
+    .ProBanner2021-module--checkitem--2d8e7 {
+      filter: brightness(0) invert(29%) sepia(7%) saturate(1782%) hue-rotate(196deg) brightness(97%) contrast(88%);
+    }
+
+    .Logo-module--logoImg--1ee7e {
+        filter: invert(24%) sepia(95%) saturate(2507%) hue-rotate(217deg) brightness(102%) contrast(92%);
+     }
+   }
+      
+  & when (@lookup = frappe) {
+    path,
+    .DeeplLogoText-module--logoText--34a4f,
+    .ProBanner2021-module--lockitem--febe3,
+    .FlyoutMenuButton-module--flyoutMenuButton--97ae6::after,
+    #top-navi-sidemenu-opener,
+    .pageFooterV2-module--socialMediaLinks--71305,
+    .ProBanner2021-module--checkitem--2d8e7 {
+      filter: brightness(0) invert(79%) sepia(9%) saturate(884%) hue-rotate(193deg) brightness(106%) contrast(92%);
+    }
+
+    .Logo-module--logoImg--1ee7e, .deeplLogo-module--logo--ffd7a {
+       filter: invert(60%) sepia(77%) saturate(428%) hue-rotate(192deg) brightness(101%) contrast(87%);
+     }
+   }
+
+  & when (@lookup = macchiato) {
+    path,
+    .DeeplLogoText-module--logoText--34a4f,
+    .ProBanner2021-module--lockitem--febe3,
+    .FlyoutMenuButton-module--flyoutMenuButton--97ae6::after,
+    #top-navi-sidemenu-opener,
+    .pageFooterV2-module--socialMediaLinks--71305,
+    .ProBanner2021-module--checkitem--2d8e7 {
+        filter: brightness(0) invert(75%) sepia(22%) saturate(286%) hue-rotate(191deg) brightness(110%) contrast(92%);
+    }
+
+    .Logo-module--logoImg--1ee7e, .deeplLogo-module--logo--ffd7a {
+        filter: invert(61%) sepia(75%) saturate(404%) hue-rotate(190deg) brightness(98%) contrast(95%);
     }
+  }
 
-    .lmt__glossary_ad_checkmark {
-      border-bottom-color: @green;
-      border-right-color: @green;
+  & when (@lookup = mocha) {
+    path,
+    .DeeplLogoText-module--logoText--34a4f,
+    .ProBanner2021-module--lockitem--febe3,
+    .FlyoutMenuButton-module--flyoutMenuButton--97ae6::after,
+    #top-navi-sidemenu-opener,
+    .pageFooterV2-module--socialMediaLinks--71305,
+    .ProBanner2021-module--checkitem--2d8e7 {
+        filter: brightness(0) invert(86%) sepia(4%) saturate(1588%) hue-rotate(192deg) brightness(101%) contrast(91%);
     }
 
-    & when (@zenMode = 1) {
-      header,
-      [class="w-full bg-white px-8 lg:px-[70px]"],
-      [class="p-8 lg:px-[70px]"],
-      footer,
-      [href="/write"].block {
-        display: none;
-      }
-
-      body {
-        background: @mantle;
-      }
-
-      [class="relative pr-0 false"] {
-        height: 100%;
-      }
-
-      [data-testid="translator"] {
-        place-content: center center;
-        align-items: center;
-
-        > div:first-of-type {
-          width: 100vw;
-        }
-      }
+    .Logo-module--logoImg--1ee7e, .deeplLogo-module--logo--ffd7a {
+        filter: invert(61%) sepia(85%) saturate(375%) hue-rotate(189deg) brightness(98%) contrast(100%);
+     }
```

Comment:
Please don't do this. Add a variable to the palette at the bottom like we do with other userstyles, and add then add a variable declaration at the top below the others. I didn't get to this PR in time so please do this in a followup PR.

Reply:
E.g. https://github.com/search?q=repo:catppuccin/userstyles+%22filter:+@%22&type=code.

Reply:
> Please don't do this. Add a variable to the palette at the bottom like we do with other userstyles, and add then add a variable declaration at the top below the others. I didn't get to this PR in time so please do this in a followup PR.

i did,  #1109